application/common/mail/mfa-enabled.html.php

```diff
@@ -26,4 +26,18 @@
 ?>
 <p>Dear <?= yHtml::encode($displayName) ?>,</p>
 
-<h2>TODO: Compose contents of this email message</h2>
\ No newline at end of file
+<p>
+    Congratulations! You have enabled 2-Step Verification on your <?= yHtml::encode($idpDisplayName) ?> Identity account.
+    This is a great way to protect access your account as well keep bad guys out of our corporate systems.
```

Comment:
Typo: `protect access your account` --> `protect access to your account`

Reply:
Done